configure.py

```diff
@@ -0,0 +1,63 @@
+import os, platform, subprocess
+
+def checkCommand(shell):
+    if platform.system() == "Windows":
+        return os.system("where " + shell)
+    else:
+        return os.system("which " + shell)
+
+def checkAll():
+    print("Checking for valid bash")
+    if checkCommand("bash"):
+        print("bash not found")
+        print("You will want a POSIX compatible environment")
+        exit(1)
+    else:
+        print("bash checked")
+    
+    print("Checking for rst2html")
+    if checkCommand("rst2html"):
+        print("rst2html not found")
+        print("Install with: pip3 install rst2html")
+        exit(1)
+    else:
+        print("rst2html checked")
+
+    print("Checking for python2")
+    if checkCommand("python2"):
+        print("python2 not found. ML still relies on it.")
+        exit(1)
+    else:
+        print("python2 works")
+
+    print("Checking for pip2")
+    if checkCommand("pip2"):
+        print("pip2 not found. ML still relies on it.")
+        exit(1)
+    else:
+        print("python2 works")
+        py1 = subprocess.check_output(["python2", "--version"])
+        py2 = subprocess.check_output(["python", "--version"])
+        if py1 == py2:
+            print("python is python2")
+        else:
+            print("You will need to route python to python2, you can install 'python-is-python2'")
```

Comment:
I thought I fixed this by removing all plain "python" usage, making all calls in the build system python2 or python3, and/or making all #!/usr/bin/env lines explicit python2 or python3.

Notably, my system doesn't have "python", and it works fine, so it will fail this configure check.

Reply:
Resolving: unnecessary req check for python being python2 was removed